pymacaroons/macaroon.py

```diff
@@ -21,7 +24,9 @@ def __init__(self,
                  identifier=None,
                  key=None,
                  caveats=None,
-                 signature=None):
+                 signature=None,
+                 version=MACAROON_V2):
```

Comment:
I wonder if we should default this to MACAROON_V1, otherwise this will break all the current
clients of pymacaroons that haven't implemented support for V2 format everywhere.